### DISCUSSION TITLE
Possibility to use Filebase as an storage solution

### DISCUSSION THREAD

**Discussion by katriellucas** [pocketbase/pocketbase#953] [Q&A] [answered]
[Filebase](https://filebase.com/) is Decentralized Storage with a compatible S3 API, it gives 5GB for free + what I believe to be 5GB of bandwidth as well. Does anyone tried to put this to work with PocketBase?

**Comment by EverlyScott** [comment]
Id assume that any S3 compatible storage solution works. I'm using Contabo Object Storage and that works perfectly fine.

**Comment by ganigeorgiev** [comment, marked as answer]
I've never used Filebase, but based on their [S3 compatibility api list](https://docs.filebase.com/getting-started/getting-started-guides/s3-api-getting-started-guide#supported-api-methods), it should work out-of-the-box with our current implementation.

From https://docs.filebase.com/getting-started/getting-started-guides/s3-api-getting-started-guide, it seems that your endpoint should be https://s3.filebase.com/, and the other settings (bucket, region, access key and secret) probably you'll find somewhere in their dashboard.
